Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2805,7 +2805,7 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, bool supportsNonP
 
     auto endArrayLayer = checkedSum<uint32_t>(descriptor.baseArrayLayer, descriptor.arrayLayerCount);
     if (endArrayLayer.hasOverflowed() || endArrayLayer.value() > arrayLayerCount())
-        return ERROR_STRING(@"endArrayLayer is not valid");
+        return ERROR_STRING([NSString stringWithFormat:@"endArrayLayer(%u) is not valid. Base texture array count is %u", endArrayLayer.value(), arrayLayerCount()]);
```

Comment:
This was added because I ran into this error bringing up the red triangle and the error message was not sufficient descriptive to understand what was wrong without debugging